CONTRIBUTING.md

```diff
@@ -90,7 +90,7 @@ please include a note in your commit message explaining why.
 
 ## Communication
 
-We use [Slack](https://www.ory.sh/chat). You are welcome to drop in and ask
+We use [discord](https://www.ory.sh/chat). You are welcome to drop in and ask
```

Comment:
Why are you returning this back? I fixes this in #412.

`https://www.ory.sh/chat` redirects to Slack.

Reply:
Ahh, my bad - we use an automation script for this and your changes haven't been added there. I'll fix that